packages/angular/package.json

```diff
@@ -21,7 +22,6 @@
     "@angular-eslint/template-parser": "~14.4.0",
     "@angular/animations": "^14.3.0",
     "@angular/cli": "^14.2.13",
-    "@angular/common": "^14.3.0",
```

Comment:
Removing `"@angular/common"` (and `"rxjs"` below) doesn't feel quite right to me. Can you explain why we would remove them?

Reply:
See https://github.com/aws-amplify/amplify-ui/pull/5796#discussion_r1759233733